.github/workflows/esti.yaml

```diff
@@ -1144,4 +1144,122 @@ jobs:
                 body: output
               })
             }
-  
\ No newline at end of file
+
+  e2e-readonly:
+    name: E2E - Readonly Repository
+    needs: 
+      - deploy-image
+      - login-to-amazon-ecr
+    runs-on: ubuntu-latest-8-cores
+    timeout-minutes: 20
+    services:
+      dynamodb:
+        image: amazon/dynamodb-local:1.13.6
+      lakefs:
+        image: ${{ needs.login-to-amazon-ecr.outputs.registry }}/lakefs:${{ needs.deploy-image.outputs.tag }}
+        credentials:
+          username: ${{ needs.login-to-amazon-ecr.outputs.docker_username }}
+          password: ${{ needs.login-to-amazon-ecr.outputs.docker_password }}
+        ports:
+          - '8000:8000'
+        env:
+          LAKEFS_DATABASE_TYPE: dynamodb
+          LAKEFS_DATABASE_DYNAMODB_ENDPOINT: http://dynamodb:8000
+          LAKEFS_DATABASE_DYNAMODB_AWS_ACCESS_KEY_ID: AKIAIOSFDNN7EXAMPLEQ
+          LAKEFS_DATABASE_DYNAMODB_AWS_SECRET_ACCESS_KEY: wJalrXUtnFEMI/K7MDENG/bPxRfiCYEXAMPLEKEY
+          LAKEFS_BLOCKSTORE_TYPE: local
+          LAKEFS_EMAIL_SUBSCRIPTION_ENABLED: true
+          LAKEFS_AUTH_ENCRYPT_SECRET_KEY: some random secret string
+          LAKEFS_STATS_ENABLED: false
+          LAKEFS_LOGGING_LEVEL: DEBUG
+    steps:
+      - uses: actions/checkout@v4
+      - uses: actions/setup-node@v3
+        with:
+          node-version: 18
+          cache: "npm"
+          cache-dependency-path: webui/package-lock.json
+      - name: Install dependencies
+        working-directory: webui
+        run: npm ci
+      - name: Get installed Playwright version
+        id: playwright-version
+        working-directory: webui
+        run: echo "PLAYWRIGHT_VERSION=$(node -e "console.log(require('./package-lock.json').dependencies['@playwright/test'].version)")" >> $GITHUB_ENV
+      - name: Cache Playwright binaries
+        uses: actions/cache@v3
+        id: playwright-cache
+        with:
+          path: |
+            ~/.cache/ms-playwright
+            ~/.cache/ms-playwright-*
+          key: ${{ runner.os }}-playwright-${{ env.PLAYWRIGHT_VERSION }}
+          restore-keys: |
+            ${{ runner.os }}-playwright-
+      - name: Install browsers (no cache)
+        if: steps.playwright-cache.outputs.cache-hit != 'true'
+        working-directory: webui
+        run: npx playwright install --with-deps chromium
+      - name: Install browsers (with cache)
+        if: steps.playwright-cache.outputs.cache-hit == 'true'
+        working-directory: webui
+        run: npx playwright install-deps chromium
+      - name: Playwright run
+        env:
+          BASE_URL: http://localhost:8000
+        working-directory: webui
+        run: npx playwright test --project=readonly
```

Comment:
Could this at least be a matrix with all the other e2e tests?

Reply:
I've merged the new tests back into the `common` Playwright project, so no need for two jobs. Removed.